style.css

```diff
@@ -637,11 +642,15 @@ a.current{
 }
 
 .product-box .image{
-    border:1px solid black;
     height:250px;
     margin:7% 9% 3%;
 }
 
+.product-box .image img{

```

Comment:
Although I resized most of my images to 250px by 250px, some photos were slightly smaller/larger so by having this line of code, it ensures all images are the same size anyway.